HEC.FDA.Model/paireddata/GraphicalUncertainPairedData.cs

```diff
@@ -200,12 +202,26 @@ public IPairedData SamplePairedData(double probability, bool computeIsDeterminis
                 y[i] = _StageOrLogFlowDistributions[i].InverseCDF(probability);
             }
             PairedData pairedData = new PairedData(_NonExceedanceProbabilities, y, _metaData);
-            if (!pairedData.IsValidPerMetadata)
```

Comment:
I misunderstood the change log. You've removed the check on isValidPerMetadata.

That means what we're actually doing here is checking that it's monotonic, and forcing it if it's not. And we're forcing strictly monotonic. That's cool with me, but is that cool with you? I seem to recall us discussing that we needed to separate out things that had to be weekly monotonic vs strictly, which was the impetus for the "IsValidPerMetadata" method.

Reply:
Because this is a frequency function, the function should be strictly monotonically increasing. IsValidPerMetadata was not catching non-monotonicity (not just weak monotonicity, but decreasing stages with decreasing AEPs), and I could not figure out why, so I just went to a different approach.